### PR TITLE
PY-18144 Support Pycharm import from zip files.

### PR DESCRIPTION
**Pull request by Ahuge**
PY-18144 Update. In response to Ilya Kazakevich's response, I have submitted my patch and created a PR.

We are falling back to importlib to load a module from within a zip file. Imp does not support this.

### CODE REVIEW

**Comment by klikh** [comment]
We've received the CLA. Thanks.

File: python/helpers/pycharm/pycharm_run_utils.py [hunk 1]
```diff
@@ -30,8 +30,15 @@ def adjust_django_sys_path():
 def import_system_module(name):
   if sys.platform == "cli":    # hack for the ironpython
       return __import__(name)
-  f, filename, desc = imp.find_module(name)
-  return imp.load_module('pycharm_' + name, f, filename, desc)
+  try:
+    f, filename, desc = imp.find_module(name)
+    return imp.load_module('pycharm_' + name, f, filename, desc)
+  except:
```
Comment (throwable-one) [inline python/helpers/pycharm/pycharm_run_utils.py]: Is it possible to except certain error?

**Comment by Ahuge** [comment]
Updated with a more specific exception clause.

Should I be logging the exception that I am catching?

**Comment by Ahuge** [comment]
What do I need to do to get this pulled in?

**Comment by wangjoshuah** [comment]
BUMP

**Pull request closed without merging**